charts/synthetics-job-manager/templates/deployment.yaml

```diff
@@ -74,6 +74,9 @@ spec:
             - name: CHECK_TIMEOUT
               value: {{ .Values.global.checkTimeout | quote }}
             {{- end }}
+
+            - name: HEAVYWEIGHT_WORKERS
+              value: {{ include "synthetics-job-manager.runtimeParallelism" . | quote }}
```

Comment:
is that extra dot before the pipe suppose to be there?

Reply:
Yep! It's part of the Include command. There are a couple examples on lines 55 & 56 of this file